features/00-permissions.feature

```diff
@@ -22,7 +22,6 @@ When I touch the Background Location Services row
 Then Calabash should dismiss the alert
 And background location services are authorized
 
-@skip_for_non_english
```

Comment:
I believe we also need to remove `Before("@skip_for_non_english")` in `features/support/01_launch.rb` file

Reply:
Done!